Makefile

```diff
@@ -0,0 +1,11 @@
+# $FreeBSD$
+
+TESTSDIR=	${LOCALBASE}/tests/tcptestsuite
+ATF_TESTS_SH=	tcptestsuite_atf_test
+TEST_METADATA=	required_programs="packetdrill"
+CLEANFILES+=	tcptestsuite_atf_test.sh
+
+tcptestsuite_atf_test.sh:
```

Comment:
This is missing a dependency on `make_atf.sh`; if `tcptestsuite_atf_test.sh` existed already and you modified `make_atf.sh`, it would not rebuild `tcptestsuite_atf_test.sh`.

Reply:
It's also missing dependencies on essentially the entire directory tree, but I suppose I'll fix this easy one.

Reply:
...in commit 7640b1e.